shared/trainer/defaults.yaml

```diff
@@ -2,7 +2,7 @@ output_path: /network/tmp1/schmidtv/yb_runs/test_v1
 # -------------------
 # -----  Tasks  -----
 # -------------------
-tasks: [a, d, h, s, t, w]
+tasks: [a, d, h, s, t, w, m]
```

Comment:
can you remove all references to `w` and WaterDecoder?

Reply:
Done (as far as I can tell)